Swift Vocabulary/Swift Vocabulary/View Controllers/WordsTableViewController.swift

```diff
@@ -0,0 +1,90 @@
+//
+//  WordsTableViewController.swift
+//  Swift Vocabulary
+//
+//  Created by James Sedlacek on 5/19/20.
+//  Copyright © 2020 James Sedlacek. All rights reserved.
+//
+
+import UIKit
+
+class WordsTableViewController: UITableViewController {
+
+    var words: [VocabularyWord] = [VocabularyWord(word: "Variable", definition: "Variable Definition"),
+    VocabularyWord(word: "Constant", definition: "Constant Definition"),
+    VocabularyWord(word: "Function", definition: "Function Definition")
+    ]
```

Comment:
Prefer indenting, with the elements line up with one of these two styles:

``` swift
var words: [VocabularyWord] = [
    VocabularyWord(word: "Variable", definition: "Variable Definition"),
    VocabularyWord(word: "Constant", definition: "Constant Definition"),
    VocabularyWord(word: "Function", definition: "Function Definition")
]
```
or

``` swift
var words: [VocabularyWord] = [VocabularyWord(word: "Variable", definition: "Variable Definition"),
                               VocabularyWord(word: "Constant", definition: "Constant Definition"),
                               VocabularyWord(word: "Function", definition: "Function Definition")]
```